hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/TestS3AProxy.java

```diff
@@ -20,18 +20,17 @@
 
 import java.io.IOException;
 
-import com.amazonaws.ClientConfiguration;
-import com.amazonaws.Protocol;
 import org.assertj.core.api.Assertions;
 import org.junit.Test;
+import software.amazon.awssdk.http.apache.ProxyConfiguration;
 
 import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.s3a.impl.AWSClientConfig;
 import org.apache.hadoop.test.AbstractHadoopTestBase;
 
 import static org.apache.hadoop.fs.s3a.Constants.PROXY_HOST;
 import static org.apache.hadoop.fs.s3a.Constants.PROXY_PORT;
 import static org.apache.hadoop.fs.s3a.Constants.PROXY_SECURED;
-import static org.apache.hadoop.fs.s3a.S3AUtils.initProxySupport;
 
 /**
```

Comment:
@mehakmeet I noticed we introduced a new config option `fs.s3a.proxy.ssl.enabled`, but the port of the proxy is still set using `fs.s3a.connection.ssl.enabled` [here](https://github.com/mehakmeet/hadoop/blob/8d7a7f74e62e4e37a91dc800cfbca698518ec37f/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AUtils.java#L1323). Was wondering why need the new `fs.s3a.proxy.ssl.enabled` option? can we not use `fs.s3a.connection.ssl.enabled` to set the protocol too? Asking here as this SDK V2 branch uses `fs.s3a.connection.ssl.enabled` to set both protocol + port, but that causes this test to fail as it uses `fs.s3a.proxy.ssl.enabled`.

Reply:
fixed on this branch yes, don't believe it's been fixed on trunk yet.